src/nvim/ui_client.c

```diff
@@ -59,7 +58,10 @@ uint64_t ui_client_start_server(int argc, char **argv)
 #ifdef MSWIN
     os_open_conin_fd();
 #else
-    dup(stderr_isatty ? STDERR_FILENO : STDOUT_FILENO);
+    int result = dup(stderr_isatty ? STDERR_FILENO : STDOUT_FILENO);
```

Comment:
can just use `(void)` to ignore this. if neither fd is usable we will never get here.